.ci/buildAll.sh

```diff
@@ -7,7 +7,7 @@ ORIG_PWD="$PWD"
 
 for APP_PATH in $APP_PROJECTS; do
     cd $APP_PATH
-    ./gradlew build
+    ./gradlew assembleDebug
```

Comment:
Why? `build` builds all build types (and thus depends on `assembleDebug`), plus it does run checks, if any, which is exactly what we should do in CI, IMO.